DetDescr/python/HcalGeometry.py

```diff
@@ -116,38 +119,61 @@ def make_v13(self):
         """
         self.v13 = HcalReadoutGeometry()
 
+        self.v13.num_sections = 5
+        self.v13.num_layers = [100, 28, 28, 26, 26]
         self.v13.scint_thickness = 20.0
         self.v13.scint_width = 50.0
+
+        back_scint_length = 3100.0
+        side_TB_scint_length = back_scint_length - 600. #TODO/FIXME Dummy value
+        side_LR_scint_length = back_scint_length - 800. #TODO/FIXME Dummy value
```

Comment:
I think it should be:
```
28*(20+20+4)+600 = 1832
```
based on:
```
sideTB_layers*side_layer_thick+side_Ecal_dy
```
https://github.com/LDMX-Software/ldmx-sw/blob/trunk/Detectors/data/ldmx-det-v13/hcal.gdml#L22
and
https://github.com/LDMX-Software/ldmx-sw/blob/trunk/Detectors/data/ldmx-det-v13/hcal.gdml#L181